tools/perfscale-audit/metric-client/metric-client.go

```diff
@@ -49,9 +49,22 @@ const (
 //
 //	at all times.
 const (
-	vmiPhaseCount = `sum by (phase) (kubevirt_vmi_phase_count{})`
+	vmiPhaseCount                 = `sum by (phase) (kubevirt_vmi_phase_count{})`
+	avgVirtAPICPUUsage            = `avg(sum(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-api.*", container!="",container!="POD"}[%ds]))by (pod))`
+	avgVirtAPIMemUsageInMB        = `avg(avg_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds]))/1024/1024`
+	minVirtAPIMemUsageInMB        = `min_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds])/1024/1024`
+	maxVirtAPIMemUsageInMB        = `min_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds])/1024/1024`
+	avgVirtControllerCPUUsage     = `avg(sum(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-controller.*", container!="",container!="POD"}[%ds])) by (pod))`
+	avgVirtControllerMemUsageInMB = `max(avg_over_time(container_memory_rss{pod=~"virt-controller.*", container!="POD", container!=""}[%ds]))/1024/1024`
+	minVirtControllerMemUsageInMB = `min_over_time(container_memory_rss{pod=~"virt-controller.*", container!="POD", container!=""}[%ds])/1024/1024`
+	maxVirtControllerMemUsageInMB = `min_over_time(container_memory_rss{pod=~"virt-controller.*", container!="POD", container!=""}[%ds])/1024/1024`
```

Comment:
What about virt-handler?

Reply:
The virt-api and virt-controller usages will immediately help with kwok testing in #12117. This work is part of that. 

Having virt-handler metrics will be nice but since virt-handlers are spread across multiple nodes how do we accurately aggregate the information and show it in benchmarks is not immediately clear. For that reason can we treat virt-handler metrics in a issue?

Reply:
Sounds good.